### ISSUE TITLE
All Joystick indexes are always connected on Win 8.1 and Win 10

### ISSUE THREAD

**Issue by endimilojkoski** [mono/MonoGame#4020]
Joystick.GetCapabilities(X).IsConnected or Joystick.GetState(X).IsConnected 
are always true even though there are no Joysticks connected.

This is not the case on Win 7 and Win 8, just on Win 8.1 and Win 10.

Happening in WindowsGL project.

**Comment by endimilojkoski** [comment]
I have done more research on this and the issue is not consistent.
I am trying on my Win 7, Win 8 and also Win 10 PCs and there are no issues with Joystick, everything works perfectly fine!
I also added new logic that let users add Joysticks in game and it works great.
My game (http://rainingblobs.com) now supports up to 8 Joysticks and 2 players can play on keyboard.

2 of my friends have the above reported problem on Windows 8.1 but the game runs fine other than that.
But my 3rd friend has Win 10 and have the above problem but also have some additional game breaking problems that no one had so far.
He gets Index out of range exceptions somewhere around the Joystick code in game.
Basically, the game sees that there are Joyticks connected and tries to check if start button or any other buttons are pressed on given screens and that is when I get the index out of range exception and the game breaks.
The same is happening for my two friends with Win 8.1 but it does not throw index out of range exception.

For safety until this is fully resolved I added config file option that disables Joystick discovery and utilization in the game and the game works fine with it for my 3rd friend with Win 10 that had the game breaking bugs.
